CHANGELOG.md

```diff
@@ -353,6 +355,8 @@
   [#9225](https://github.com/Kong/kong/pull/9225)
 - Add `so_keepalive` to listen options (e.g. `KONG_PROXY_LISTEN`)
   [#9225](https://github.com/Kong/kong/pull/9225)
+- Add LMDB dbless config persistence
```

Comment:
```suggestion
- Add LMDB dbless config persistence and removed the JSON based
  config cache for faster startup time
```

Reply:
the config cache removal is currently in the breaking changes section. Do you want I add it here too?